src/components/QuotesLoader/index.tsx

```diff
@@ -86,22 +86,23 @@ const useStyles = makeStyles({
     },
 });
 
-const Loader = React.memo(function ({ show }) {
+const Loader = React.memo(function ({ show }: { show: boolean }) {
     const classes = useStyles();
     const isBigScreen = useBigScreen();
     const shuffled = useMemo(() => [...quotes], []);
     const [index, setIndex] = useState(0);
-    const intervalId = useRef(null);
 
     useEffect(() => {
         if (show) {
             shuffleArray(shuffled);
-            intervalId.current = setInterval(
+            const intervalId = setInterval(
```

Comment:
why isn't the ref needed?

Reply:
https://stackoverflow.com/questions/45862713/clearinterval-in-react